application/routes.py

```diff
@@ -13,11 +13,15 @@ def create_routes(api):
     from .controllers.tags_controller import TagsController
     from .controllers.presentations_controller import PresentationsController
     from .controllers.teams_controller import TeamsController
+    from .controllers.users_controller import UsersController
 
     add_method(api, '/', "show_status", HomeController, method="get")
     add_resource(api, '/images', ArtifactsController)
     add_method(api, '/images', "update_many", ArtifactsController, method="patch")
     add_resource(api, '/teams', TeamsController)
+    add_resource(api, '/users', UsersController)
+    add_method(api, '/users/login', "login", UsersController, method="post")
+    add_method(api, '/users/logout', "logout", UsersController, method="post")
```

Comment:
Do these have to work like this. Didn't we want to make these routes restful with an `/authentication` endpoint that you can post and delete to?